### ISSUE TITLE
Unexpected `Nones` in result of `ITree.find_at`

### ISSUE THREAD

**Issue by NiklasBeierl** [mikpom/rbi_tree#1]
Hello, 
Thanks for providing this package, I would love to use it, tho I cant explain away the behavior I am seeing here:
```python
>>> for i, int in enumerate([(1,3),(5,9),(6,10)]):
...     t.insert(*int, i)
...
>>> t.find_at(2)
[(1, 3, 0), None]
>>> t.find_at(6)
[(5, 9, 1), None, (6, 10, 2), None]
```
Why is the output peppered with `None`s?

**Comment by IridiumXOR** [comment]
Maybe the problem is here 
https://github.com/mikpom/rbi_tree/blob/3443171f64f8e566e7c03bbaa05030ea1aaa5353/rbi_tree/tree.pyx#L134
due to the double append (maybe a bad copy/paste :) )

**Comment by mikpom** [comment]
Thanks for this bug report, should be fixed in b41fd8cca9090c5ec1d1a603c497280b0bff7ea8 and updated pypi version.
